src/backend-query.js

```diff
@@ -2,17 +2,17 @@
 import hetmechDefinitions from './definitions.json';
 // url for metagraph (node and edge metatypes and abbreviations)
 const metagraphUrl =
-  'https://raw.githubusercontent.com/hetio/hetionet/master/hetnet/json/hetionet-v1.0-metagraph.json';
+  'https://raw.githubusercontent.com/hetio/hetionet/0009918ab3ecfe15ef8aeb9437f117e72d96f668/hetnet/json/hetionet-v1.0-metagraph.json';
 // url for hetio definitions (metanodes, properties, etc)
 const hetioDefinitions =
-  'https://raw.githubusercontent.com/hetio/hetionet/master/describe/definitions.json';
+  'https://raw.githubusercontent.com/hetio/hetionet/84aa810d4af88befc0d96d49e325a5c1a77f5995/describe/definitions.json';
```

Comment:
Should we make all three hetionet commits the same for ease? The most recent commit should okay

Reply:
I can do that. I just went through a picked the latest commit for each file. I'll change it.